pymisp/mispevent.py

```diff
@@ -522,6 +522,17 @@ def add_attribute_tag(self, tag, attribute_identifier):
         if not attribute:
             raise Exception('No attribute with identifier {} found.'.format(attribute_identifier))
         return attribute
+    
+    
+    def get_attribute_tags(self, attribute_identifier):
+        attribute = None
+        for a in self.attributes:
+            if (a.id == attribute_identifier or a.uuid == attribute_identifier or
+                        attribute_identifier == a.value or attribute_identifier in a.value.split('|')):
+                return a.Tag
+        if not attribute:
```

Comment:
Attribute will always be None, so if we reach this line, we have no hits so it will always be true and raise the exception.
And I prefer returning an empty array than an exception.

Reply:
Good point! Works for me as well.